src/classes/RD2_RecurringDonation.cls

```diff
@@ -77,20 +77,31 @@ public inherited sharing class RD2_RecurringDonation {
     * @param scheduleService RD Schedule Service
     * @return RD2_RecurringDonation This record instance
     */
-    public RD2_RecurringDonation reviseNextDonationDate(RD2_ScheduleService scheduleService) {  
+    public RD2_RecurringDonation reviseNextDonationDate(RD2_ScheduleService scheduleService) {
         return reviseNextDonationDate(scheduleService, rd.RecurringDonationSchedules__r);
     }
 
+    @TestVisible
+    private static PS_IntegrationServiceConfig config {
```

Comment:
- The `PS_IntegrationServiceConfig` should be the property of the `RD2_ElevateIntegrationService` (see PR #5338  integration [class](https://github.com/SalesforceFoundation/NPSP/blob/11bd49b951785fe3a9658432ea6e6d26e09bbe75/src/classes/RD2_ElevateIntegrationService.cls) for more detail).
- The `RD2_OpportunityEvaluationService` should have an instance of `RD2_ElevateIntegrationService`   which should provide answer if the integration is enabled or not (new method in the `RD2_ElevateIntegrationService` ) .